### PR TITLE
Wsp migrate docs

### PR DESCRIPTION
**Pull request by f0ssel**

### CODE REVIEW

**Comment by f0ssel** [comment]
[ch15124]

File: admin/satellites/migrate-to-satellites.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+---
+title: Migrate to satellite deployments
+description:
```
Comment (deansheather) [inline admin/satellites/migrate-to-satellites.md:3]:
```suggestion
description: |
```

File: admin/satellites/migrate-to-satellites.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+Although no down time is expected, users may experience negative impacts latency
+during the migration until all steps are completed for the region.
+
+## Deploy satellites
+
+The first step will be to deploy new satellite deployments in each region you
+wish to expand into. Follow the steps outlined in
+[satellite management](./management.md) for creating a Satellite deployment. It
+is recommended to deploy the satellite into a different namespace from the
+existing workspace provider. The access URL for these deployments will be used
```
Comment (deansheather) [inline admin/satellites/migrate-to-satellites.md:55]: It doesn't have to exist in the same namespace or cluster as the workspaces

File: admin/satellites/migrate-to-satellites.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+`australia-southeast1` region.
+
+## Enable NetworkingV2
+
+Navigate to **Manage** > **Providers** and select the workspace provider. Enable
+the NetworkingV2 toggle and save.
+
+Rebuild a workspace and ensure connectivity to the workspace before moving onto
+the next steps. Note that latency to the workspace may be negatively impacted in
+this step until users are connecting to the new replica deployments.
```
Comment (deansheather) [inline admin/satellites/migrate-to-satellites.md:68]:
```suggestion
this step until users are connecting to the new satellite deployments.
```

File: admin/satellites/migrate-to-satellites.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+Developers will always have the lowest latency possible by connecting to the
+Coder deployment closed to them geographically. Therefore, the users in
+Australia should always connect to the `coder-sydney.example.com` deployment.
+
+## Annotate Kubernetes resources
+
+The old envproxy-based workspace provider deployment is no longer in use.
+However the Kubernetes ServiceAccount, Role, and RoleBindings created by the
+helm chart are still required. To safely remove the unused helm chart, we will
+add an annotation to the resources we'd like to keep first.
```
Comment (deansheather) [inline admin/satellites/migrate-to-satellites.md:81]: Maybe put a warning saying if you don't follow this step the workspace provider might die

**Review by deansheather — COMMENTED** [review]
LGTM

**Review by khorne3 — APPROVED** [review]

**Pull request merged**